x/stakeibc/keeper/reward_converter.go

```diff
@@ -103,12 +103,23 @@ func (k Keeper) CheckForCommunityPoolRebate(
 			"unable to calculate rebate amount for %s since total delegations are 0", chainId)
 	}
 
+	// It also shouldn't be possible for the liquid stake amount to be greater than the full TVL
+	if rebateInfo.LiquidStakeAmount.GT(hostZone.TotalDelegations) {
```

Comment:
yeah good catch - although I think we'd want to avoid registering the rebate until it's been delegated (otherwise the rewards don't technically belong to the CP yet)

Also, if we switch to using stDYDX for the liquid staked amount, this should no longer be able to error in normal conditions

Reply:
> we'd want to avoid registering the rebate until it's been delegated (otherwise the rewards don't technically belong to the CP yet)

ah yeah that makes sense 

> Also, if we switch to using stDYDX for the liquid staked amount, this should no longer be able to error in normal conditions

because we'd compare liquidStakedAmount to stTokenSupply instead of to TotalDelegations?

Reply:
yeah exactly

Reply:
Makes sense. Resolving.